main.js

```diff
@@ -24,9 +24,9 @@ app.get('/', function (req, resp) {
 
 // ************************************* Enter Page *********************************************
 /* 1. Check for free lots
-*  2. Increase number of vehicles in garage by 1
-*  3. Create new ticket for vehicle
-*/
+ *  2. Increase number of vehicles in garage by 1
+ *  3. Create new ticket for vehicle
+ */
 app.get('/enter', function (req, resp) {
     connection.beginTransaction(function (err) {
         if (err) {
```

Comment:
`resp.send` does not stop code execution as it might seem it does.  Make sure you either `return` immediately after the response or simply `return resp.send(..)`

Reply:
As well as any other `response` instances of this file, this will cause the node process to throw a stack trace in some situations. Which it does in your example application if you have no running mysql database.